src/utils/feature.flag.ts

```diff
@@ -1,15 +1,8 @@
 /**
  * Feature flags will be determined by environment variables and all environment variables in nodejs are
- * either string or undefined. This function will ensure that 'false', '0', 'off' etc remain falsy
+ * either string or undefined.
+ * This function will ensure that 'true', '1', 'on' will be Truthy
  */
-export const isActiveFeature = (flag: string | undefined): boolean => {
-
-  if (flag === undefined) {
-    return false;
-  }
-  const featureFlag = flag.toLowerCase();
-  return !(featureFlag === "false" ||
-    featureFlag === "0" ||
-    featureFlag === "off" ||
-    featureFlag === "");
+export const isActiveFeature = (flag: string): boolean => {
+  return ['1', 'true', 'on'].includes(flag.toString().toLowerCase());
```

Comment:
the original version had a check for undefined, do we still need that check as undefined.toString() will throw error

Reply:
Just passing an empty string as default value in the flag argument will have improved readability and coverage